src/redux/books/books.js

```diff
@@ -0,0 +1,30 @@
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
+const initialState = [];
+
+export const addBook = (book) => ({
+  type: ADD_BOOK,
+  payload: book,
+});
+
+export const removeBook = (id) => ({
+  type: REMOVE_BOOK,
+  payload: id,
+});
```

Comment:
Your application is not performing as expected, I cannot add books. When I click on the `add book` button nothing happens. Kindly fix this. As a result of this, I couldn't test your `remove book` function.

Reply:
Hello reviewer, That is the functionality that should be added in the next milestone when we link the react and redux.
We are not supposed to complete that action at this branch.